mediorum/server/replicate.go

```diff
@@ -203,3 +205,33 @@ func (ss *MediorumServer) pullFileFromHost(host, cid string) error {
 
 	return ss.replicateToMyBucket(cid, resp.Body)
 }
+
+func (ss *MediorumServer) moveFromDiskToMyBucket(diskPath string, key string, checkIsMetadata bool) error {
+	source, err := os.Open(diskPath)
+	if err != nil {
+		return err
+	}
+	defer source.Close()
+
+	// ignore this file if it's a metadata file
+	if checkIsMetadata {
+		bytes, err := io.ReadAll(source)
+		if err != nil {
+			return err
+		}
+		var jsonData map[string]interface{}
+		err = json.Unmarshal(bytes, &jsonData)
+		if err == nil {
+			// if unmarshal is successful, it's a JSON/metadata file (not an image or track)
+			return nil
+		}
+	}
+
+	// write to bucket, record in blobs table that we have it, and delete the original file from disk
+	err = ss.replicateToMyBucket(key, source)
```

Comment:
do some kind of stat on the destination to make sure we have the same number of bytes